docs/reference/glossary.md

```diff
@@ -122,12 +132,22 @@ A message contains information to be delivered to interested parties during exec
 
 - [Messages](/components/concepts/messages.md)
 
+### Outbound connector
+
+Outbound [connectors](#connector) in Camunda Platform 8 allow workflows to trigger with external systems or services, making it possible to integrate workflows with other parts of a business process or system architecture.
+
 ### Partition
 
 A partition represents a logical grouping of data in a Zeebe broker. This data includes process instance variables stored in RocksDB, commands, and events generated by Zeebe stored in the log. The number of partitions is defined by configuration.
 
 - [Partitions](/components/zeebe/technical-concepts/partitions.md)
 
+### Polling
```

Comment:
we should add connector, otherwise it is confusing

Reply:
Adjusted this in my latest commit 👍